ilastik/workflows/edgeTrainingWithMulticut/edgeTrainingWithMulticutWorkflow.py

```diff
@@ -20,6 +20,7 @@
 ###############################################################################
 import argparse
 import numpy as np
+import sys
```

Comment:
```suggestion
```

Reply:
sys is actually used later in this file - I guess that import was missing